pkg/channel/attributes/dispatch_error_test.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package attributes
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+	"github.com/cloudevents/sdk-go/v2/binding"
+	"github.com/stretchr/testify/assert"
+)
+
+// Test The NewDispatchErrorExtension() Functionality
+func TestNewDispatchErrorExtension(t *testing.T) {
+
+	// Test Data
+	code := 99
+	data := []byte("TestData")
+
+	// Perform The Test
+	dispatchErrorExtension := NewDispatchErrorExtension(code, data)
+
+	// Validate Results
+	assert.NotNil(t, dispatchErrorExtension)
+	assert.Equal(t, code, dispatchErrorExtension.Code)
+	assert.Equal(t, data, dispatchErrorExtension.Data)
+}
```

Comment:
Can you add a test with data bigger than 1024?

Reply:
Will do - thanks for the suggestion!

Reply:
Also - any thoughts on 1024 as a MAX value?  I just pulled that out of thin air - maybe it's too large?

Reply:
Nah i think it's the right amount